charts/govuk-jobs/templates/content-store-mongo-to-postgres-cronjob.yaml

```
@@ -0,0 +1,65 @@
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: "content-store-mongo-to-postgres-cron"
+  labels:
+    app: "content-store-mongo-to-postgres-cron"
+    app.kubernetes.io/component: "content-store-mongo-to-postgres-cron"
+spec:
+  schedule: "15 5 * * *"  
+  jobTemplate:
+    metadata:
+      name: "content-store-mongo-to-postgres-cron"
+      labels:
+        app: "content-store-mongo-to-postgres-cron"
+        app.kubernetes.io/component: "content-store-mongo-to-postgres-cron"
+    spec:
+      backoffLimit: 0
+      template:
+        metadata:
+          name: "content-store-mongo-to-postgres-cron"
+          labels:
+            app: "content-store-mongo-to-postgres-cron"
+            app.kubernetes.io/component: "content-store-mongo-to-postgres-cron"
+        spec:
+          enableServiceLinks: false
+          securityContext:
+            fsGroup: {{ .Values.securityContext.runAsGroup }}
+            runAsNonRoot: {{ .Values.securityContext.runAsNonRoot }}
+            runAsUser: {{ .Values.securityContext.runAsUser }}
+            runAsGroup: {{ .Values.securityContext.runAsGroup }}
+          restartPolicy: Never
+          volumes:
+            - name: mongo-export
+              emptyDir: {}
+          initContainers:
+            - name: export-mongo-data
+              image: "content-store:release"
+              imagePullPolicy: "Always"
+              command: ["bundle exec rake"]
```

`["bundle exec rake"]` means "[exec](https://manpages.ubuntu.com/manpages/jammy/en/man3/exec.3.html) a binary called `bundle exec rake` with no arguments", i.e. it's equivalent to running `bundle\ exec\ rake` or `'bundle exec rake'` in the shell. To run the command you intended you'd need to write `["bundle", "exec", "rake"]` (quotes optional).

You don't need the `bundle exec` though (because we're in a container, so there's only ever going to be one Ruby app there by design) so this is preferred:

```suggestion
              command: ["rake"]
```

Similarly below.